shfmt.patch

```diff
@@ -0,0 +1,87 @@
+--- scripts/setup.sh.orig
```

Comment:
This file logically belongs to the .sh lint commit.

Reply:
Oops

Reply:
This file is still present in the 1st commit and then later deleted in the 3rd commit. Please remove it from the 1st commit.

Reply:
Done